### PR TITLE
Fixes bug 1156594 - Add star to searched events

### PR DESCRIPTION
**Pull request by bugzPDX**

### CODE REVIEW

File: airmozilla/search/templates/search/home.html [hunk 1]
```diff
@@ -92,6 +92,7 @@ <h5 class="entry-upcoming">
 
           <a class="go" href="{{ href }}">{{ _('See more') }}</a>
         </p>
+        <a class="star{% if star_on %} star-on{% endif %}" data-id="{{ event.id }}" title="Save as a starred event"></a>
```
Comment (peterbe) [inline airmozilla/search/templates/search/home.html]: That `{% if star_on %}` would that ever be true? 
Isn't that just for when you display events on the starred page?

**Comment by peterbe** [comment]
r+

**Comment by peterbe** [comment]
Would you mind making it 1 commit?

**Comment by peterbe** [comment]
Also, can you add a test to `search/tests/test_views.py` that checks that the link is in there?

**Comment by bugzPDX** [comment]
Yep I will do that now!

**Comment by peterbe** [comment]
I'll merge this as soon as Travis gets itself unstuck.

**Pull request merged**